content/en/docs/contribute/participate/roles-and-responsibilities.md

```diff
@@ -70,7 +70,7 @@ After submitting at least 5 substantial pull requests and meeting the other
    [sponsor](/docs/contribute/advanced#sponsor-a-new-contributor) your
    membership.
 
-   Ask for sponsorship in the [#sig-docs channel on Slack](https://kubernetes.slack.com) or on the
+   For example, if your contribution is more related to documentation，you can ask for sponsorship in the [#sig-docs channel on Slack](https://kubernetes.slack.com) or on the
```

Comment:
Rather than this clarification, we could add the more general clarification that I suggest in https://github.com/kubernetes/website/issues/32046

There's related work from SIG ContribEx to make the general contributor guide more accessible / visible.

Reply:
I agree with you.